x-pack/legacy/plugins/siem/public/components/timeline/body/stateful_body.tsx

```diff
@@ -150,6 +203,13 @@ const StatefulBodyComponent = React.memo<StatefulBodyComponentProps>(
       [id]
     );
 
+    // Sync to timelineTypeContext.selectAll so parent components can select all events
+    useEffect(() => {
+      if (timelineTypeContext.selectAll) {
```

Comment:
For My Information, can use that later on also to unselect all if we are putting it back the value to false

Reply:
Yeah, it's on the provider of the TimelineTypeContext prop to `StatefulEventViewer` that's responsible for resetting it, so there's some flexibility. I've made a note to better document this in the TimelineTypeContext docs as this is a good one to know about. Thanks for mentioning! 🙂